routes/views/job.js

```diff
@@ -8,34 +8,35 @@ exports = module.exports = function (req, res) {
 	// Set locals
 	locals.section = 'jobs';
 	locals.filters = {
-		post: req.params.post,
+		job: req.params.job,
 	};
 	locals.data = {
-		posts: [],
+		jobss: [],
```

Comment:
@maryam-olabisi 'jobss' here seems like a typo

Reply:
Yes, thank you.